package.json

```diff
@@ -23,23 +25,30 @@
   },
   "dependencies": {
     "moment": "^2.10.6",
-    "react": "^0.13.3",
-    "react-bootstrap": "^0.26.1",
-    "react-datepicker": "^0.13.0",
-    "react-time-select": "^1.0.2"
+    "react": "^0.14.2",
```

Comment:
Module versions should really be locked down, unless you believe there's a reason not to? We lock down version numbers to ensure that if a newer version of a module were to be released, we can be sure it would not affect our modules. Not everyone respected subversion backwards compatibility, unfortunately. @shackpank ?